exampleFiles/tasks.json

```diff
@@ -7,7 +7,7 @@
             "label": "Run Hollywood Main script",
             "type": "shell",
             "command": "${config:hollywood.exePath}",
-            "args": ["${workspaceFolder}/${config:hollywood.mainFile}"],
+            "args": ["$${config:hollywood.mainFile}"],
```

Comment:
Ooops, there are two `$` signs! :-) -> `$${config:hollywood.mainFile}`

Reply:
Sorry about that, fixed it with a new commit :)

Reply:
No problem. :-) Thanks a lot for your work!